library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -3346,6 +3346,8 @@
     <string name="labs_enable_session_manager_summary">Have greater visibility and control over all your sessions.</string>
     <string name="labs_enable_client_info_recording_title">Enable client info recording</string>
     <string name="labs_enable_client_info_recording_summary">Record the client name, version, and url to recognise sessions more easily in session manager.</string>
+    <string name="labs_enable_voice_broadcast_title">Enable Voice broadcast (under active development)</string>
```

Comment:
@Florian14 feel free to replace the "Voice" with "voice". It would be more adapted to the other flags